frontend/src/components/Table.tsx

```diff
@@ -0,0 +1,51 @@
+import './Table.scss';
```

Comment:
`Table.scss` is empty (for now) but will likely end up with custom styling (not just yet)